docs/user/quickstart.rst

```diff
@@ -0,0 +1,139 @@
+Quickstart Guide
+================
+
+.. contents:: **Table of contents**:
+    :depth: 2
+    :local:
+
+Requirements
+------------
+
+- Devices running at least OpenWrt 12.09 Attitude Adjustment, older
+  versions of OpenWrt have not worked at all in our tests
+- Devices must have enough free RAM to be able to upload the new image to
+  ``/tmp``
+
+1. Create a Category
+--------------------
+
+Create a category for your firmware images by going to *Firmware
+management > Firmware categories > Add firmware category*, if you use only
+one firmware type in your network, you could simply name the category
+"default" or "standard".
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-firmware-upgrader/docs/docs/images/quickstart-category.gif
+    :target: https://raw.githubusercontent.com/openwisp/openwisp-firmware-upgrader/docs/docs/images/quickstart-category.gif
+
+If you use multiple firmware images with different features, create one
+category for each firmware type, eg:
+
+- WiFi
+- SDN router
+- LoRa Gateway
+
+This is necessary in order to perform mass upgrades only on specific
+firmware categories when, for example, a new *LoRa Gateway* firmware
+becomes available.
+
+2. Create the Build Object
+--------------------------
+
+Create a build a build object by going to *Firmware management > Firmware
```

Comment:
`s/a build a build/a build/`

Reply:
Thank you for spotting this. I have fixed it!